src/jit/compiler.cpp

```diff
@@ -4359,6 +4359,7 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
         bool doCopyProp      = true;
         bool doAssertionProp = true;
         bool doRangeAnalysis = true;
+        int  iterations      = 1;
```

Comment:
The macros that define the config COMPLUS_ settings use signed int, I'm more inclined to match them than to make this unsigned.  With the change to a while loop, specifying a negative repeat count would then be the same as specifying zero.